src/lib/y2storage/partition_tables/msdos.rb

```diff
@@ -0,0 +1,17 @@
+require "y2storage/storage_class_wrapper"
+require "y2storage/partition_tables/base"
+
+module Y2Storage
+  module PartitionTables
+    # A MBR partition table
+    #
+    # This is a wrapper for Storage::Msdos
+    class Msdos < Base
+      include StorageClassWrapper
```

Comment:
I see a lot of repetition there for this module, maybe better way is to add it to Base or have new class StorageBased which contain it. Or is it used somewhere without base or vice versa? for me it looks coupled together.